tests/ontopy_tests/test_patch.py

```diff
@@ -1,48 +1,91 @@
 """Tests Owlready2 patches implemented in ontopy/patch.py
 
-Implemented as a script, such that it easy to understand and use for debugging.
 """
+
 import pytest
 
 from ontopy import get_ontology
 
 from owlready2 import owl, Inverse
 
+from utilities import setassert
+
+
+def test_get_by_label_onto(emmo: "Ontology") -> None:
```

Comment:
What version of EMMO does the `emmo` argument correspond to? Does it correspond to a stable tag or will it change with time?

Reply:
Ok. Since the pytest fixtures are so opaque, it might be good to add a comment saying this (since we might have to update the tests when get_ontology() is updated to return a newer version of EMMO).

Reply:
I do not agree that they are opaque, but to make it more readily clear I added a comment in the emmo fixture on conftest.